src/main/java/eisenwave/elytra/command/ElytraToggleCommand.java

```diff
@@ -31,7 +31,7 @@ public ElytraToggleCommand(final SuperElytraPlugin plugin, final PlayerManager p
 
     @Override
     public boolean onCommand(final CommandSender commandSender, final Command command, final String s,
-                             final String[] args) {
+            final String[] args) {
```

Comment:
Seems like a lot of auto-formatter changes in this PR that don't match what mine is doing. Can you revert these so the changes are the smallest set possible to do your changes?